docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+Since DID messages are always to be signed and the 'chain of custody' of the signing key can be traced throughout the identity lifespan, replay, message insertion, deletion, modification and man-in-the-middle attacks are prevented.
+
+### Stateless Identities
+
+Unlike for-purpose Blockchains or Smart contract based DID methods, the IOTA tangle does not track and store the state of DID Document. This prevents dusting attacks against the nodes. However, the client that resolves the DID Document has the responsibility to always validate the entire history of the DID Document as to validate the 'chain of custody' of the signing keys. If a client does not do this, identity hijacking is easily possible. The IOTA Identity framework, which implements the `did:iota` method, provides an easy to use client-side validation implementation. 
+
+### Snapshotting
+
+IOTA allows feeless data and value messages. As such it has a large history of messages combining to several TBs at the time of writing. Most nodes 'snapshot' (forget/delete) older transactions, while they keep the UTXOs stored in the ledger. The snapshot settings are local, therefore all nodes may store a different length of history. As such, older DID messages would not be avaliable at every node. Since the entire history of the DID Document is required for client-side validation, this may become problematic. It is currently recommended to either use your own node, that does not snapshot or use the [Chronicle Permanode](https://github.com/iotaledger/chronicle.rs), which keeps the entire history of the Tangle. This is not a longterm scalable solution, which is why other methods for either keeping state or selective storage is considered. 
+
+### Denial of Service Attacks
+
+There is little to no barrier for users to send any message on the Tangle. While this is a great feature of IOTA, it has its drawbacks in terms of spam. Anyone can post messages on the indexation of an Identity and therefore hinder the query speed. To reduce a users own load on their identity verification speed, the Diff chain was introduced. Other messages that are spammed are identified and dropped as early as possible in the resolution process without triggering any errors or warnings. These are, for example, random data messages or incorrectly signed DID messages. 
+
+IOTA nodes provide pagination for more then 100 messages on an indexation, requiring multiple queries for identities with more messages. Since the queries are the bottleneck in the resolution process, it is recommended to run an IOTA node in a nearby location if fast verification is required on a physical location. Verification may, in the future, also be outsourced to nodes directly, to reduce the effect from spam attacks. If an identity is signficantly spammed, it might be best to create a new identity. 
```

Comment:
indexation -> index

ctrl+f "indexation" - replace all - "index"

> IOTA node in a nearby location if fast verification is required on a physical location

rewrite this

the effect from spam -> the effect of spam

> If an identity is signficantly spammed, it might be best to create a new identity. 

:(